releasenotes/notes/enable-transport-for-metadata-calls-727408e9f5916cb7.yaml

```diff
@@ -0,0 +1,14 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Use the default agent transport for host metadata calls.
+    This allows usage of the config `no_proxy` setting for host metadata calls.
+    By default metadata IP's are added to the transport's `no_proxy` list.
+    Added config flag `proxy.use_proxy_for_host_metadata` to disable this behavior. 
```

Comment:
Is there a use case where we would want to go through the proxy for this ?

Reply:
Unsure of a specific case. However the intention here is to allow the flexibility to use the proxy if someone wanted to.